Makefile

```diff
@@ -1,10 +1,26 @@
-PROMBENCH_CMD        = ./prombench
-DOCKER_TAG = docker.io/prombench/prombench:2.0.0
+PROMBENCH_CMD := ./prombench
+DOCKER_REPO := prombench
+DOCKER_IMAGE_NAME := prombench
+DOCKER_IMAGE_TAG := 2.0.0
 
+include Makefile.common
+
+# This is to prevent go get promu, staticcheck & govendor from updating go.mod 
+export GO111MODULE = off
```

Comment:
wouldn't it be better to modify the original `Makefile.common` file? IIUC you'd never want for any project that `go get {promu,statitccheck,govendor}` modifies the local `go.mod` so the corresponding `go get` commands in `Makefile.common` should be prefixed with `GO111MODULE=off`.